### PR TITLE
Single Vanities - Add to Cart & Your Cart Page test -NT

### PR DESCRIPTION
**Pull request by nursena-tekoglu**
Add to cart and your cart page functionality test is created. Test runs successfully. Screenshots are attached. @KubraSA 

Screenshot 2023-11-05 at 23 45 34
Screenshot 2023-11-05 at 23 45 58
Screenshot 2023-11-05 at 23 46 13
Screenshot 2023-11-05 at 23 46 28

### CODE REVIEW

**Comment by nursena-tekoglu** [comment]
Once main is updated I will merge the main to my branch, and push the codes to PR again. @KubraSayli20

File: cypress/integration/step_definitions/Ui/single_vanities.cy.js [hunk 1]
```diff
@@ -31,3 +31,125 @@ When('User clicks to Single Vanities', () => {
...
+    const addedProduct = '.cart-item'; 
+    cy.get(addedProduct).should('exist');
+});
+
+And ('user clicks to Check-Out Button' , () => {
+    singleVanitiesPage.clickCheckOutButton()
+})
+
+Then ('it should direct to the Payment' , () => {
+    const expectedTitle = 'Paiement';
```
Comment (KubraSA) [inline cypress/integration/step_definitions/Ui/single_vanities.cy.js:153]: What about using this common function here: [https://github.com/abeyis/Avaluxury/blob/38c2d844ab8a1c86230f3ff74aafe2c12996e605/cypress/integration/step_definitions/Ui/commonSteps.js#L9](url)

File: cypress/integration/step_definitions/Ui/single_vanities.cy.js [hunk 2]
```diff
@@ -31,3 +31,125 @@ When('User clicks to Single Vanities', () => {
...
+})
+
+And ('User clicks to random add-to-cart button' , () =>{
+    singleVanitiesPage.addToCartRandomProduct() 
+})
+
+And('it should direct to Your Cart page', () => {
+
+    const expectedTitle = 'Your Shopping Cart'; 
+    cy.title().should('contain', expectedTitle);
```
Comment (KubraSA) [inline cypress/integration/step_definitions/Ui/single_vanities.cy.js:139]: we have common function this. shared the link below.

**Comment by nursena-tekoglu** [comment]
Test is updated according the comments. 

Tests are passing. @KubraSA 

Screenshot 2023-11-08 at 23 03 13

File: cypress/integration/step_definitions/Ui/single_vanities.cy.js (hunk 1 above)
Comment (nursena-tekoglu) [reply cypress/integration/step_definitions/Ui/single_vanities.cy.js]: Done. @KubraSA

File: cypress/integration/step_definitions/Ui/single_vanities.cy.js (hunk 2 above)
Comment (nursena-tekoglu) [reply cypress/integration/step_definitions/Ui/single_vanities.cy.js]: Done. @KubraSA

**Pull request merged**